src/main/groovy/com/bmuschko/gradle/docker/DockerJavaApplicationPlugin.groovy

```diff
@@ -30,7 +32,7 @@ import org.gradle.util.ConfigureUtil
  */
 class DockerJavaApplicationPlugin implements Plugin<Project> {
     static final String COPY_DIST_RESOURCES_TASK_NAME = 'dockerCopyDistResources'
-    static final String DOCKERFILE_TASK_NAME = 'dockerDistTar'
+    static final String DOCKERFILE_TASK_NAME = 'dockerfile'
```

Comment:
Want me to create the a No-op task with the old name and just make it depend on this one?